localstack/utils/testutil.py

```diff
@@ -762,3 +694,51 @@ def response_arn_matches_partition(client, response_arn: str) -> bool:
         == boto3.session.Session().get_partition_for_region(parsed_arn["region"])
         and client.meta.partition == parsed_arn["partition"]
     )
+
+
+def create_zip_file(file_path, zip_file=None, get_content=False, content_root=None, mode="w"):
```

Comment:
I'm assuming we're not using this function from the CLI, so safe to leave it in here..?

Reply:
Exactly, this is merely used in some integrations tests!